packages/terra-form-fieldset/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ ChangeLog
 Unreleased
 ----------
 
+### Changed
+* Removed DefaultFieldset.test.jsx
```

Comment:
could we add a reason :) it seems weird to only we removed a test

Reply:
I wasn't given a reason :(

Reply:
@AlexBrizi I think its because we aren't using this test examples in our tests so its an extra, unused example. You will need to check the test urls we use in the test spec for wdio.

Reply:
On the [issue](https://github.com/cerner/terra-core/issues/1843), @yuderekyu mentioned, "There is a form-fieldset test example that isn't tested in the spec. We can remove this."

Reply:
Updated changelog comment noting the file was unnecessary: https://github.com/cerner/terra-core/pull/1922/commits/edae2b9f027b540d50b0dfe10dadc468f243c529